scijava-ops-opencv/src/main/java/module-info.java

```diff
@@ -30,6 +30,8 @@
 	exports org.scijava.ops.opencv;
 
 	requires java.scripting;
+	requires org.scijava.ops.api;
```

Comment:
Figure out why this is needed

Reply:
It's needed because of the tests. But I got rid of it by adding an argument to surefire within the POM